contracts/p0/Main.sol

```diff
@@ -21,23 +25,23 @@ contract Pausable is Ownable, IPausable {
         _;
     }
 
-    function pause() external {
```

Comment:
Is there a reason these had to become public?

Reply:
Finally I used external for all and set the Owner and Pauser from the deployer as agreed.